modules/system/assets/js/snowboard/utilities/Sanitizer.js

```diff
@@ -0,0 +1,58 @@
+import Singleton from '../abstracts/Singleton';
+
+export default class Sanitizer extends Singleton {
```

Comment:
Where is this plugin added?

Reply:
Utilities are included through the main Snowboard class (method `loadUtilities`).